docs/detections/session-view.asciidoc

```diff
@@ -84,4 +93,37 @@ Session View includes an additional badge not pictured above:
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=80%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+
+beta::[]
+
+Terminal output view helps provide a more complete understanding of commands executed by users or adversaries, and assists with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and shell commands (if the shell was in echo mode).
+
+To enable Terminal output data capture:
+. Go to *Manage* -> *Policies*, then select one or more of your {elastic-defend} integration policies to edit.
+. On the *Policy settings* tab, scroll down to the Linux event collection section near the bottom of the page
+and select the *Include session data* and *Capture terminal output* options.
+
+You can configure several additional settings by clicking *Advanced settings* at the bottom of the page:
+
+* Select the maximum number of kilobytes of output to record from a single process by modifying `linux.advanced.tty_io.max_kilobytes_per_process`. Default: 512 KB. For example, with the default value, a process' output will not be recorded after the 512th KB.
+* Select the maximum number of kilobytes of output to send to {es} as a single event by modifying `linux.advanced.tty_io.max_kilobytes_per_event`. Default: 512 KB. For example, with the default value, the maximum amount of output that will be sent to {es} as a single event is 512 KB. Additional data will be captured as a new event.
+* Select the maximum duration in seconds for which output is batched by modifying `linux.advanced.tty_io.max_event_interval_seconds`. Default: 30 seconds. For example, with the default value, any recorded output will be sent to {es} every 30 seconds (unless it first exceeds the `max_kilobytes_per_event` value, in which case it might be sent sooner).
+
+[role="screenshot"]
+image::images/session-view-output-viewer.png[Terminal output view,width=100%,height=100%]
+
+1. The search bar. Use to find and highlight search terms within the current session.
+The left and right arrows allow you to navigate through search results.
+2. The right-side scroll bar. Use along with the bottom scroll bar to navigate output data that doesn't fit on a single screen.
+3. The playback controls: *Start*, *Prev*, *Play*, *Next*, *Last*, and the progress bar. Use to advance or rewind the session's commands and output. Click anywhere on the progress bar to jump to that part of the session. The marks on the bar represent processes that generated output. Click them or the *Prev* and *Next* buttons to skip between processes.
+4. The *Fit screen*, *Zoom in*, and *Zoom out* buttons. Use to adjust the text size. Use Session view's *Fullscreen* button (located next to the *Close session viewer* button) to better fit output with long lines, such as for graphical programs like `vim`.
```

Comment:
The last sentence about the **Fullscreen** button is kind of confusing because that button isn't even in the screenshot above. I think I get why it's here, but maybe reformatting as a Tip might work better, since then it'll be separate from the rest of the description of # 4 from the diagram.